scripts/checks

```diff
@@ -29,7 +29,13 @@ fi
 echo "+++ golint tree..."
 for i in $(find . -maxdepth 1 -type d -name '*' | grep -vE '/\..*$')
 do
-  golint "${i}"
+  output=$(golint "${i}") # golint has no exit codes
+  if [ -n "${output}" ]
+  then
+    echo -e "!!! golint checking failed: "
+    echo -n "${output}"
+    exit 1
```

Comment:
Just a minor nit. This will cause the script to error out on first module it hits errors, would it instead be better to collect all go-lint errors and print them together?